wiki/Installation/macOS/ko.md

```diff
@@ -0,0 +1,20 @@
+# macOS
+
+This page will roughly tell you how to get osu! working on your macOS device.
+
+## Minimum system requirements
+
+- A decent computer running macOS.
+
+## Installing osu!
+
+The macOS version of osu! is experimental. The game may not function properly or even boot at all.
+
+1. Go to the unofficial [download page](https://osu.ppy.sh/community/forums/topics/682197) and download the installer.
+2. Locate the installer, right click, and select open.
+   - Double left clicking on the file may not open it, since macOS may recognize the app as developed by an unidentified developer. Right clicking and selecting open will solve this problem.
+3. The installer will automatically run. The installation location cannot be changed and will be installed directly to `osu!.app`. Moving `osu.app` will move all of the files that come with it.
```

Comment:
I wonder if the wiki needs an unofficial installation method on macOS.
And some of the content seems incorrect. (For example, there are two expressions in this line: `osu!.app` and `osu.app`)

@cl8n

Reply:
I'm not sure, I don't know anything about macOS. this was all added by one person in 8d6f0759fb326beab5fc2303a6154abca72bfeff

Reply:
All right, I'll have to make some modifications.

Reply:
should be osu!.app